src/language/ast.d.ts

```diff
@@ -204,6 +204,16 @@ export interface DocumentNode {
   readonly kind: 'Document';
   readonly loc?: Location;
   readonly definitions: ReadonlyArray<DefinitionNode>;
+  readonly comments: ReadonlyArray<CommentNode>;
```

Comment:
Please `comments` optional: `comments?: ReadonlyArray<CommentNode>` and remove empty arrays from manually created document nodes in unrelated tests (e.g. `buildASTSchema-test.js`)